static/app/components/replays/canvasReplayerPlugin.tsx

```diff
@@ -102,45 +213,59 @@ export function CanvasReplayerPlugin(events: eventWithTime[]) {
      */
     handler: async (
       e: eventWithTime,
-      _isSync: boolean,
+      isSync: boolean,
       {replayer}: {replayer: Replayer}
     ) => {
-      if (
-        e.type === EventType.IncrementalSnapshot &&
-        e.data.source === IncrementalSource.CanvasMutation
-      ) {
-        const source = replayer.getMirror().getNode(e.data.id);
-        const target =
-          canvases.get(e.data.id) ||
-          (source && cloneCanvas(e.data.id, source as HTMLCanvasElement));
-
-        // No canvas found for id... this isn't reliably reproducible and not
-        // exactly sure why it flakes. Saving as metric to keep an eye on it.
-        if (!target) {
-          Sentry.metrics.increment('replay.canvas_player.no_canvas_id');
-          return;
+      // isSync = =true means it is fast forwarding vs playing
+      // nothing to do when fast forwarding since canvas mutations for us are
+      // image snapshots and do not depend on past events
```

Comment:
```suggestion
      // isSync = true means it is fast forwarding vs playing
      // nothing to do when fast forwarding since canvas mutations for us are
      // image snapshots and do not depend on past events
```

Reply:
This is a big fix too, previously we were always running the canvas mutations regardless of `isSync` - now we skip them when user fast forwards.